### ISSUE TITLE
WARNING:tensorflow:From ./hangul_model.py:219: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.

### ISSUE THREAD

**Issue by markstur** [IBM/tensorflow-hangul-recognition#27]
WARNING:tensorflow:From ./hangul_model.py:219: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See @{tf.nn.softmax_cross_entropy_with_logits_v2}.
